ci/test-conformance-gke.sh

```diff
@@ -29,12 +29,13 @@ MODE="report"
 
 RUN_ALL=true
 RUN_SETUP_ONLY=false
+SETUP_TEST_ONLY=false
 RUN_CLEANUP_ONLY=false
 TEST_FAILURE=false
 
 _usage="Usage: $0 [--cluster-name <GKEClusterNameToUse>]  [--kubeconfig <KubeconfigSavePath>] [--k8s-version <ClusterVersion>] \
                   [--svc-account <Name>] [--user <Name>] [--gke-project <Project>] [--gke-zone <Zone>] [--log-mode <SonobuoyResultLogLevel>] \
-                  [--svc-cidr <ServiceCIDR>] [--host-type <HostType] [--gloud-path] [--setup-only] [--cleanup-only]
+                  [--svc-cidr <ServiceCIDR>] [--host-type <HostType] [--gloud-path] [--setup-only] [--setup-test-only] [--cleanup-only]
```

Comment:
do you think we still need a configurable `ServiceCIDR` and the `GKE_SERVICE_CIDR` variable? I think the main motivation was being able to set it correctly in the Antrea Agent config, which is no longer needed. That being said, there is no harm in keeping it around.

Reply:
The `GKE_SERVICE_CIDR` variable is passed in during gcloud clusters create:
```
${GCLOUD_PATH} container --project ${GKE_PROJECT} clusters create ${CLUSTER} --image-type ${GKE_HOST} \
    --cluster-version ${K8S_VERSION} --zone ${GKE_ZONE} \
    --enable-ip-alias --services-ipv4-cidr ${GKE_SERVICE_CIDR}
```
This is actually customizable according to https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--cluster-ipv4-cidr
@jianjuns do you think there's still a need for us to expose this param?

Reply:
I'll let Jianjun comment, but since the script is meant to be used to test Antrea, there isn't too much value in keeping the parameter IMO. We also don't have an equivalent for EKS / AKS. I think we should remove `GKE_SERVICE_CIDR` from gke-installation.md as well, to simplify installation steps.

Reply:
Got rid of the service cidr as requested